src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ColorSupplier supplier = new ColorSupplier();
+        Color randomColor = supplier.getRandomColor();
+        int randomNumber = random.nextInt(100) + 1;
```

Comment:
magic numbers, make it a constant value

Reply:
not fixed, extract 100 to a class level constant